packages/store-sync/src/recs/recsStorage.test.ts

```diff
@@ -0,0 +1,47 @@
+import { describe, expect, it } from "vitest";
+import { blockLogsToStorage } from "../blockLogsToStorage";
+import { recsStorage } from "./recsStorage";
+import { createWorld, getComponentEntities, getComponentValue } from "@latticexyz/recs";
+import superjson from "superjson";
+// TODO: figure out if this needs to be imported from a package rather than relative path
+// TODO: if relative imports are fine, probably should put the e2e test data with its mud config?
+import mudConfig from "../../../../e2e/packages/contracts/mud.config";
+import testLogsJson from "../../../../test-data/world-logs.json";
+import { groupLogsByBlockNumber } from "@latticexyz/block-logs-stream";
+import { StoreEventsLog } from "../common";
+import { singletonEntity } from "./singletonEntity";
+
+const testLogs = superjson.deserialize<readonly StoreEventsLog[]>(testLogsJson as any);
```

Comment:
superjson wants a more generic type while the import has a specific type, so we cast to `any`

<img width="654" alt="image" src="https://github.com/latticexyz/mud/assets/508855/819d2a59-f2cc-4e7c-93ff-04b67baca4b5">

I wanted to cast as `SuperJSONResult` but superjson doesn't export this, and it wouldn't let me cast to `ReturnType<typeof superjson.serialize>` (effectively `SuperJSONResult`)

Reply:
ended up moving to rpc logs instead, which are json serializable